python/python/ert_gui/simulation/models/ensemble_smoother.py

```diff
@@ -18,6 +18,7 @@ def setAnalysisModule(self, module_name):
 
     def runSimulations(self, arguments):
         prior_context = self.create_context( arguments )
+        active_realizations = self.count_active_realizations( prior_context )
```

Comment:
A better check would be to *start* with 
```python
self.checkHaveSufficentRealisations( active_realisations )
```
before even starting. That test should fail at once if you try the process which is described in the PR description.